### ISSUE TITLE
[performance] CPU is idle even when there are operations ready to be executed

### ISSUE THREAD

**Issue by yann-yy** [tensorflow/tensorflow#25923]
<em>Please make sure that this is a bug. As per our [GitHub Policy](https://github.com/tensorflow/tensorflow/blob/master/ISSUES.md), we only address code/doc bugs, performance issues, feature requests and build/installation issues on GitHub. tag:bug_template</em>

**System information**
- Have I written custom code (as opposed to using a stock example script provided in TensorFlow): Yes
- OS Platform and Distribution (e.g., Linux Ubuntu 16.04): CentOS 7.5
- Mobile device (e.g. iPhone 8, Pixel 2, Samsung Galaxy) if the issue happens on mobile device: No
- TensorFlow installed from (source or binary): source
- TensorFlow version (use command below): 1.12.0
- Python version: 2.7
- Bazel version (if compiling from source): 0.19.2- (@non-git)
- GCC/Compiler version (if compiling from source): 4.8.5
- CUDA/cuDNN version: No
- GPU model and memory: No

You can collect some of this information using our environment capture [script](https://github.com/tensorflow/tensorflow/tree/master/tools/tf_env_collect.sh)
You can also obtain the TensorFlow version with
python -c "import tensorflow as tf; print(tf.GIT_VERSION, tf.VERSION)"

**Describe the current behavior**
I'm trying to feed tensors produces by a ParseExample OP when inference an exported graph to avoid 
serialization/deserialization tf record of input examples.
I have got the correct prediction results. But the performance is not consistent with my expectation.
There is several idle time in the timeline. And the feed tensors are not processing in parallel.

**Describe the expected behavior**
No CPU idle time in prediction. No ParseExample OP in inference should be faster.

**Code to reproduce the issue**
Provide a reproducible test case that is the bare minimum necessary to generate the problem.
`saved_bundle.session->Run(run_options, input_tensor_pairs, {output_tensor_name}, {}, &output, nullptr);`
**Other info / logs**
Include any logs or source code that would be helpful to diagnose the problem. If including tracebacks, please include the full traceback. Large logs and files should be attached.
chrome tracing results:
with ParseExample, no cpu idle during inference
![image](https://user-images.githubusercontent.com/32092715/53081857-a7216300-3536-11e9-807c-791c1f47f4c2.png)
[with_parse_example.json.txt](https://github.com/tensorflow/tensorflow/files/2883952/with_parse_example.json.txt)
without ParseExample, cpu idle during inference with red marks
![image](https://user-images.githubusercontent.com/32092715/53082111-231bab00-3537-11e9-922f-3ab8c2f5f743.png)
[without_parse_example.json.txt](https://github.com/tensorflow/tensorflow/files/2883961/without_parse_example.json.txt)

Any advices will be appreciated, Thanks

**Comment by yann-yy** [comment]
Is there any progress on this issue?

**Comment by yann-yy** [comment]
I think this issue could be closed as is not active for a long time.

**Comment by bw4sz** [comment]
@yann-yy did you end making headway? I'm seeing alot idle CPU and not alot of info about what causes this (IO bound?)

**Comment by sushreebarsa** [comment]
@yann-yy  We see that you are using old version of tensorflow (1.x)which is officially considered as end of life, We recommend that you upgrade to 2.4 or later version and let us know if the issue still persists in newer versions.Please open a new issue in case you face any errors, we will get you the right help .Hence moving this to closed status.Thanks!
